### ISSUE TITLE
Why my bounding box result has no score?

### ISSUE THREAD

**Issue by francismontalbo** [AlexeyAB/darknet#6219]
Hello, everyone.

Just wanted to ask why when I get predictions it just put a bounding box with a label. But has no score provided?

Hopefully, you can help. Thank you!

**Comment by lsd1994** [comment]
Do you use the lastest repo?

**Comment by francismontalbo** [comment]
Yes, I am using the latest one. It has no score but it works well. @lsd1994

**Comment by francismontalbo** [comment]
> Do you use the lastest repo?

Yes, I am using the latest one. It has no score but it works well. @lsd1994

**Comment by lsd1994** [comment]
@francismontalbo It's normal.

**Comment by francismontalbo** [comment]
> @francismontalbo It's normal.

I see.. so there is no score when I'm using the detector for image.

**Comment by lsd1994** [comment]
You can modify source code if you want to show score.

**Comment by francismontalbo** [comment]
> You can modify source code if you want to show score.

Which source code can I modify to do this? Thank you!

**Comment by stephanecharette** [comment]
There are "helper" libraries such as DarkHelp (I'm the author) which gives you some more options when drawing the labels.  For example:

- `names_include_percentage`:  https://www.ccoderun.ca/DarkHelp/api/classDarkHelp.html#ad23b58d26aed92e80f64e1023236c4b1
- `annotation_shade_predictions`:  https://www.ccoderun.ca/DarkHelp/api/classDarkHelp.html#ab2faeea90254663d9a207fa5e008a31b

DarkHelp may be used either as a C++ library, or as a command-line tool to call from bash or python.
